rasa/importers/importer.py

```diff
@@ -0,0 +1,247 @@
+        interpreter: "NaturalLanguageInterpreter" = RegexInterpreter(),
+        template_variables: Optional[Dict] = None,
+        use_e2e: bool = False,
+        exclusion_percentage: Optional[int] = None,
+    ) -> StoryGraph:
+        return StoryGraph([])
+
+    async def get_config(self) -> Dict:
+        return await self._importer.get_config()
+
+    async def get_nlu_data(self, language: Optional[Text] = "en") -> TrainingData:
+        return await self._importer.get_nlu_data(language)
+
+
+class CombinedFileImporter(TrainingFileImporter):
+    """`TrainingFileImporter` which supports using multiple `TrainingFileImporter` as
+        if it would be a single instance.
+    """
+
+    def __init__(self, importers: List[TrainingFileImporter]):
+        self._importers = importers
+
+    async def get_config(self) -> Dict:
+        configs = [await importer.get_config() for importer in self._importers]
+        return reduce(lambda merged, other: {**merged, **(other or {})}, configs, {})
+
+    async def get_domain(self) -> Domain:
+        domains = [await importer.get_domain() for importer in self._importers]
+        return reduce(
+            lambda merged, other: merged.merge(other), domains, Domain.empty()
+        )
+
+    async def get_story_data(
+        self,
+        interpreter: "NaturalLanguageInterpreter" = RegexInterpreter(),
+        template_variables: Optional[Dict] = None,
+        use_e2e: bool = False,
+        exclusion_percentage: Optional[int] = None,
+    ) -> StoryGraph:
+        story_graphs = [
+            await importer.get_story_data(
+                interpreter, template_variables, use_e2e, exclusion_percentage
+            )
+            for importer in self._importers
+        ]
+        return reduce(
+            lambda merged, other: merged.merge(other), story_graphs, StoryGraph([])
+        )
+
+    async def get_nlu_data(self, language: Optional[Text] = "en") -> TrainingData:
+        nlu_datas = [
+            await importer.get_nlu_data(language) for importer in self._importers
+        ]
+        training_data = reduce(
+            lambda merged, other: merged.merge(other), nlu_datas, TrainingData()
+        )
+        return training_data
+
+
+class SimpleFileImporter(TrainingFileImporter):
+    """Default `TrainingFileImporter` implementation."""
+
+    def __init__(
+        self,
+        config_file: Optional[Text] = None,
+        domain_path: Optional[Text] = None,
+        training_data_paths: Optional[Union[List[Text], Text]] = None,
+    ):
+        if config_file and os.path.exists(config_file):
+            self.config = io_utils.read_config_file(config_file)
+        else:
+            self.config = {}
+
+        self._domain_path = domain_path
+
+        if training_data_paths:
+            self._story_files, self._nlu_files = data.get_core_nlu_files(
+                training_data_paths
+            )
+
+    async def get_config(self, **kwargs: Optional[Dict[Text, Any]]) -> Dict:
```

Comment:
Why did you add `**kwargs: Optional[Dict[Text, Any]]` as argument?

Reply:
legacy, removed it.